ansible/site.yml

```diff
@@ -55,3 +55,21 @@
     - swarm_agent
   tags:
     - swarm
+
+- hosts: influxdb_server
+  roles:
+    - influxdb
+  tags:
+    - influxdb
+
+- hosts: all
```

Comment:
specifying `hosts: all` will cause it be installed on localhost when we run with digital ocean inventory and it will fail. Instead use `hosts: nodes`